webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -0,0 +1,157 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import './tooltip.css';
+import Octicon, {GitMerge, GitPullRequest, IssueClosed, IssueOpened} from '@primer/octicons-react';
+import ReactMarkdown from 'react-markdown';
+
+import Client from 'client';
+import {getLabelFontColor} from '../../utils/styles';
+
+export const LinkTooltip = ({href}) => {
```

Comment:
FYI: CI is failing because of this line:
```
  10:30  error  'href' is missing in props validation  react/prop-types
 ```

Reply:
I'll take a loot at it!

Reply:
Fixed, I had a typo.